### PR TITLE
Include URL in library information

### LINKED ISSUE #20: Proposal to Include URL in Library Information
I want to be able to navigate to the library repository from the app side.

### PR DESCRIPTION
**Pull request by SNQ-2001**
Closes #20 

# Usage
https://github.com/cybozu/LicenseList/assets/84154073/21d4bcf8-9ce2-4776-b886-c1d605c2282c

```swift
import SwiftUI
import LicenseList

struct ContentView: View {
    @Environment(\.openURL) private var openURL

    var body: some View {
        NavigationStack {
            List(Library.libraries) { library in
                NavigationLink(library.name) {
                    Text(library.licenseBody)
                        .padding(.horizontal, 16)
                        .navigationTitle(library.name)
                        .toolbar {
                            ToolbarItem(placement: .topBarTrailing) {
                                Button {
                                    if let url = library.url {
                                        openURL(url)
                                    }
                                } label: {
                                    Image(systemName: "link")
                                }
                            }
                        }
                }
            }
            .navigationTitle("License")
        }
    }
}
```

### CODE REVIEW

File: Sources/LicenseList/View+Extensions.swift [hunk 1]
```diff
@@ -27,4 +27,41 @@ extension View {
             }
         }
     }
+    
+    func navigationBarRepositoryAnchorLinkIfPossible(action: @escaping () -> Void) -> some View {
```
Comment (Kyome22) [inline Sources/LicenseList/View+Extensions.swift:31]: There is no need to add IfPossible to the name of this function if the same functionality can be provided regardless of iOS 14 or later.

----------

iOS 14以上でもそうでなくても同じ機能が提供できるなら、この関数の名前に`IfPossible`をつける必要はなさそうです。

**Review by Kyome22 — CHANGES_REQUESTED** [review]

**Comment by Kyome22** [comment]
I think it would be better to add usage instructions to the README.

~~~md
### Example for UIKit

```swift
import LicenseList

// in ViewController
let vc = LicenseListViewController()
vc.title = "LICENSE"

// If you want to anchor link of the repository
vc.licenseListViewStyle = .withRepositoryAnchorLink

navigationController?.pushViewController(vc, animated: true)
```

### Example for SwiftUI

```swift
import LicenseList

struct ContentView: View {
    var body: some View {
        NavigationView {
            LicenseListView()
                // If you want to anchor link of the repository
                .licenseListViewStyle(.withRepositoryAnchorLink)
                .navigationTitle("LICENSE")
        }
    }
}
```
~~~

**Comment by SNQ-2001** [comment]
@Kyome22 
Fixed!

File: Sources/LicenseList/View+Extensions.swift (hunk 1 above)
Comment (Kyome22) [reply Sources/LicenseList/View+Extensions.swift]: [a46e505](https://github.com/cybozu/LicenseList/pull/21/commits/a46e505dc8d0dfa513f4b18cbd1eaec5886a67b7) fixed

**Review by Kyome22 — APPROVED** [review]
LGTM

**Pull request merged**